snuba/settings_docker.py

```diff
@@ -4,6 +4,8 @@
 
 DEBUG = env("DEBUG", "0").lower() in ("1", "true")
 
+DEFAULT_RETENTION_DAYS = env("SENTRY_EVENT_RETENTION_DAYS", 90)
```

Comment:
nit: Can we match the name DEFAULT_RETENTION_DAYS in the environment variable - it doesn't only relate to `events`

Reply:
That would defeat the purpose a bit because then I cannot pass the variable directly. And when we remap it to a name, docker-compose passes an empty string instead of not passing it when it is not set.